src/app/(proper_react)/redesign/(public)/PlansTable.tsx

```diff
@@ -0,0 +1,558 @@
+              alt={l10n.getString(
+                "landing-premium-plans-table-feature-monitoring-plus",
+              )}
+            />
+          </Cell>
+        </Row>
+        <Row>
+          <Cell>
+            <VisuallyHidden>
+              {l10n.getString("landing-premium-plans-table-billing-label")}
+            </VisuallyHidden>
+          </Cell>
+          <Cell>
+            <div className={styles.priceCell}>
+              <p className={styles.billingPeriod}>
+                {l10n.getString("landing-premium-plans-table-billing-free")}
+              </p>
+              <p className={styles.cost}>
+                <b className={styles.price}>
+                  {l10n.getString("landing-premium-plans-table-price-free")}
+                </b>
+                <span className={styles.total} />
+              </p>
+              <Button variant="secondary" onPress={() => void signIn("fxa")}>
+                {l10n.getString("landing-premium-plans-table-cta-free-label")}
+              </Button>
+              <small className={styles.reassurance}>
+                {l10n.getString(
+                  "landing-premium-plans-table-reassurance-free-label",
+                )}
+              </small>
+            </div>
+          </Cell>
+          <Cell>
+            <div className={styles.priceCell}>
+              <div className={styles.billingPeriod}>
+                <BillingPeriodToggle
+                  onChange={(newValue) => setBillingPeriod(newValue)}
+                />
+              </div>
+              <p aria-live="polite" className={styles.cost}>
+                <b className={styles.price}>
+                  {billingPeriod === "yearly"
+                    ? l10n.getString(
+                        "landing-premium-plans-table-price-plus-annually",
+                        {
+                          monthlyPrice: priceFormatter.format(
+                            monthlyPriceAnnualBilling,
+                          ),
+                        },
+                      )
+                    : l10n.getString(
+                        "landing-premium-plans-table-price-plus-monthly",
+                        {
+                          monthlyPrice: priceFormatter.format(
+                            monthlyPriceMonthlyBilling,
+                          ),
+                        },
+                      )}
+                </b>
+                <span className={styles.total}>
+                  {billingPeriod === "yearly" && (
+                    <em className={styles.discount}>
+                      {l10n.getString(
+                        "landing-premium-plans-table-price-plus-annually-discount",
+                        {
+                          discountPercentage:
+                            ((monthlyPriceMonthlyBilling -
+                              monthlyPriceAnnualBilling) *
+                              100) /
+                            monthlyPriceMonthlyBilling,
```

Comment:
I'm going to leave this for now: ideally, the final pricing will have a value that makes this a nice round value, and I'm worried that otherwise this might be seen as misleading (even though of course in practice the difference will be negligible).